### DISCUSSION TITLE
Default pulled from DHCP?

### DISCUSSION THREAD

**Discussion by CoffeeAndBeerLex** [tryallthethings/DNS-Swapper#9] [Ideas]
Great little tool thanks!

Any chance the blue 'standard' DNS can be pulled from DHCP instead of hardcoded?   I have a couple of different PiHoles that are cycled depending on availability, would be nice for the move back to safe grabs the current.

**Comment by tryallthethings** [comment]
Hey @CoffeeAndBeerLex!
I'm currently working on implementing this. Is this what you're looking for?
![image](https://github.com/user-attachments/assets/1bc77a36-277f-4404-85d1-9abce0181de6)
